packages/examples/packages/dialogs/src/index.tsx

```diff
@@ -90,6 +90,12 @@ export const onRpcRequest: OnRpcRequestHandler = async ({ request }) => {
         },
       });
 
+    case 'showRequireScrollContent':
```

Comment:
```suggestion
    case 'showLongContent':
```

Reply:
Need to update the JSDoc too.

Reply:
Can we use this naming instead ?

Reply:
Yup, updated: [98d4964](https://github.com/MetaMask/snaps/pull/3173/commits/98d49647fb09fd9e8ec5411f554cc59f30be6d95)